cmd/gonomics/command_cache.txt

```diff
@@ -0,0 +1,255 @@
+##SourceDir:/home/runner/work/gonomics/gonomics/cmd/
```

Comment:
My understanding is that if a user only retrieves binaries and not all source code files, they still won't be able to parse all the source code and run the gonomics cmd that way, but they will be able to use this cache to run the gonomics cmd. And the cache gets updated whenever another user that has all the source code files runs the gonomics cmd and makes a commit. Is this right?